.github/workflows/radar-release-actions.yml

```diff
@@ -17,8 +17,6 @@ jobs:
           distribution: 'adopt'
       - name: Validate Gradle wrapper
         uses: gradle/wrapper-validation-action@e6e38bacfdf1a337459f332974bb2327a31aaf4b
-      - name: Accept Android Licenses
-        run: yes | sdkmanager --licenses || true
```

Comment:
do we not need to accept licenses anymore?

Reply:
These don't seem to do anything. Take a look at the [latest release Action](https://github.com/radarlabs/radar-sdk-android/actions/runs/1690860639). Output for this step always shows:

```
Run yes | sdkmanager --licenses || true
/home/runner/work/_temp/eaf05078-5c66-4000-a596-170fdf36efd6.sh: line 1: sdkmanager: command not found
yes: standard output: Broken pipe
```

We can leave it in (this same step is currently used in circleci configs) but if it is not doing anything I'd rather remove it to speed up the build a tiny bit.